src/types/interface.types.ts

```diff
@@ -1,14 +1,20 @@
 import { WalletType } from "./enum.types";
 import { WebMode } from "@algo-builder/web";
+import { Key, StateValue } from "@algo-builder/algob/build/types";
 
 export interface WalletStoreState {
 	walletKind: WalletType;
 	webMode: WebMode;
 	address: string;
 }
 
-export interface DaoID {
-	id: number | undefined;
+export interface DAO {
+	dao_id: number | undefined;
```

Comment:
How about use syntax `dao_id?: number`? Choice the best way as you refer <3

Reply:
oh yes, it completely slipped my mind, will update